tox.ini

```diff
@@ -0,0 +1,67 @@
+[tox]
```

Comment:
Since we're using Flit and the pyproject.toml notation, shouldn't this _and_ the test requirements go into pyproject.toml?  Something along the lines of this simplistic example:

```
[tool.flit.metadata.requires-extra]
test = [
    "pytest",
    "pytest-cov",
]

[tool.tox]
legacy_tox_ini = """
[tox]
envlist = py36
isolated_build = True

[testenv]
deps =
    pytest
    pytest-cov
commands = py.test --cov=barbacoa/ {posargs}
"""
```

For using Tox within pyproject.yaml: https://tox.readthedocs.io/en/latest/config.html

Looks like support for _actual_ toml support is on the way, but for now looks like we need to use `legacy_tox_ini` notation.

Reply:
oh can we put that in there?!

nice!